### PR TITLE
Implement additional IntSet methods

### PR DESCRIPTION
**Pull request by quinqu**
I deleted `intset.go` and kept `intset/intset.go`. No code was changed from the "intitial" code, just methods added

### CODE REVIEW

File: ch6/exercise6-1/intset/intset.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	var count int
+	for _, word := range s.words {
+		if word == 0 {
+			continue
+		}
+		for j := 0; j < 64; j++ {
+			if word&(1<<uint(j)) != 0 {
+				count++
+			}
+		}
```
Comment (wadells) [inline ch6/exercise6-1/intset/intset.go:75]: Earlier (2.3, 2.4) we bench marked different ways to count the number of bits set in a bitfield.  Do you remember how this compared?

Also, now that you've done things the harder way:

https://golang.org/pkg/math/bits/#OnesCount64

No need to change anything here, just drawing throughlines.

File: ch6/exercise6-1/intset/intset.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+func (s *IntSet) Remove(x int) {
+	word, bit := x/64, uint(x%64)
+	s.words[word] &^= 1 << bit
+}
+
+// Clears IntSet
+func (s *IntSet) Clear() {
+	for i := range s.words {
+		s.words[i] = 0
+	}
```
Comment (wadells) [inline ch6/exercise6-1/intset/intset.go:90]: Why do you choose to zero over `s.words = nil` -- the starting state for an IntSet?

File: ch6/exercise6-1/intset/intset_test.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
+package intset
+
+import (
+	"testing"
+)
+
+func TestLen(t *testing.T) {
+	x := &IntSet{}
+	x.Add(127)
+	x.Add(4091)
```
Comment (wadells) [inline ch6/exercise6-1/intset/intset_test.go:10]: Why 4091 here?  In https://github.com/quinqu/tgpl/pull/17 you tested with all small ints ( <=54).  Should one or the other be adjusted?

Comment (quinqu) [reply ch6/exercise6-1/intset/intset_test.go:10]: No reason, I will check this out.

File: ch6/exercise6-1/intset/intset.go (hunk 2 above)
Comment (quinqu) [reply ch6/exercise6-1/intset/intset.go:90]: I will test this out, I believe I tried something like that and Go wouldn't let me

**Pull request merged**